app/controllers/admin/overviews_controller.rb

```diff
@@ -90,6 +134,81 @@ def services_to_export
       end
     end
 
+    def service_summary(env)
+      published_service_ids = published(env).reject { |p| moj_forms_team_service_ids.include?(p.service_id) }.map(&:service_id)
+      result = []
+
+      published_service_ids.each do |id|
+        metadata = MetadataApiClient::Service.latest_version(id)
+        result << {
+          service_id: id,
+          service_name: metadata['service_name'],
+          locale: metadata['locale'],
+          page_counts: page_type_counts(metadata['pages']),
+          component_counts: component_type_counts(metadata['pages']),
+          confirmation_email_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'CONFIRMATION_EMAIL_COMPONENT_ID').present?,
+          save_and_return_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'SAVE_AND_RETURN').present?,
+          collect_information_by_email_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'SERVICE_EMAIL_BODY').present?,
+          send_to_json_api_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'SERVICE_OUTPUT_JSON_ENDPOINT').present?,
+          receive_csv_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'SERVICE_CSV_OUTPUT').present?,
+          external_start_page_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: 'production', name: 'EXTERNAL_START_PAGE_URL').present?,
+          analytics_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'GA4').present?,
+          reference_number_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'REFERENCE_NUMBER').present?,
+          payment_link_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'PAYMENT_LINK').present?
+        }
+      end
+
+      result
+    end
+
+    def page_type_counts(pages)
+      {
+        start_pages: pages.select { |p| p['_type'] == 'page.start' }.count,
```

Comment:
I believe here we can do a similar strategy, because despite not triggering DB queries, this many select and count will not be optimal.

Idea would be to obtain an array with all types of pages, and do a tally to obtain the count of each type:
Example:
```
['page.start', 'page.singlequestion', 'page.singlequestion', 'page.exit'].tally
=> {"page.start"=>1, "page.singlequestion"=>2, "page.exit"=>1}
```